### ISSUE TITLE
doesn't compile on deno@1.14.1

### ISSUE THREAD

**Issue by anagorsky** [deno-library/compress#9]
Hi!

There are errors on compile like that:

**error: TS2578 [ERROR]: Unused '@ts-expect-error' directive.
      // @ts-expect-error
      ~~~~~~~~~~~~~~~~~~~
    at https://deno.land/std@0.104.0/node/events.ts:100:7
    ...**
    
Could you please update dependencies? Should help I guess.
Thank you in advance!

**Comment by anagorsky** [comment]
I have created PR https://github.com/deno-library/compress/pull/10

**Comment by fuxingZhang** [comment]
Has been merged and released to v0.4.1
